src/audio/src/src.c

```diff
@@ -38,29 +38,23 @@
 #endif
 
 /* tracing */
-#define trace_src(__e, ...)						\
+#define trace_src(__e, ...)					\
 	trace_event(TRACE_CLASS_SRC, __e, ##__VA_ARGS__)
```

Comment:
Why this line changed?

Reply:
Yes, I did it to align `\` at the end

Reply:
@ktrzcinx I would normally suggest you to create a separate commit for this modification. Anyhow, since is just one character I fine with it, but please mention about it i the commit message so that people won't think was an unintentional change. 

Add something like this in the commit message:

While at it, also fix line continuation marker for trace_src.

Reply:
Added